lib/browser.rb

```diff
@@ -1,15 +1,22 @@
 class Browser
   def initialize(options = {})
     @found_jobs = []
-    @keywords = options[:keywords] || "ruby, rails"
+    @keywords = options[:keywords] || "product"
     @link_parser = LinkParser.new
     @url_validator = UrlValidator.new
+
+    @num_pages = ARGV[0] ? ARGV[0].to_i : 1
   end
 
   def scour_aggregator(aggregator)
     @aggregator = aggregator
-    page = browse_to_url(@aggregator.url)
-    scour_page(page)
+
+    (1..@num_pages).each do |page_id|
+      @aggregator.url = "https://nytm.org/made?list=true&page=#{page_id}"
```

Comment:
Looking again, this behavior actually breaks the design.  The aggregator follows OOP, so the assumption is that you can call ```#url``` and get whatever you need, regardless of how the particular Aggregator gets that value.  In this implementation, any aggregator other than NYTM is going to be steamrolled by pagination for NYTM, and start pulling from there instead of their own page(s).

I would suggest something like this instead, at least to start:

```
if @aggregator.multi_page
  urls = @aggregator.paged_urls(@num_pages)
  urls.each do |url|
    page = browse_to_url(url)
   ....
else
  page = browse_to_url(@aggregator.url)
  ...
end

```

That feels a little clunky, but do you see what I'm getting at?  As far as clunkiness, it'd be better if there was no need to do the check for ```multi_page```, but I'm not sure right now what that would look like